setup-r/src/installer.ts

```diff
@@ -373,13 +373,14 @@ async function acquireRMacOS(version: IRVersion): Promise<string> {
   // Older R versions on newer macOS cannot create a symlink to R and
   // Rscript, we'll need to do it manually.
   try {
+    await exec.exec("brew", ["unlink", "r"]); //remove homebrew R from the PATH
```

Comment:
Can you pls put this in its own `try {}` block, so we create the symlinks, even if homebrew `r` was not installed? Thanks!